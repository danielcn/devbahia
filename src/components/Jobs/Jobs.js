import React, {Component} from 'react'

class Jobs extends Component {
  render () {
    return (
      <div className="Jobs">
      </div>
    );
  }
}

export default Jobs;
