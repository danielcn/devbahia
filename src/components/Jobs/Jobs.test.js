import React from 'react';
import { shallow } from 'enzyme';
import Jobs from './Jobs';

describe('Jobs Snapshot', () => {
  test('renders', () => {
    const jobs = shallow(<Jobs />);
    expect(jobs).toMatchSnapshot();
  });
});
