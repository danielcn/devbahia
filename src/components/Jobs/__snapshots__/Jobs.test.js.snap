// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jobs Snapshot renders 1`] = `ShallowWrapper {}`;
